index.js

```diff
@@ -1,3 +1,9 @@
+// Ensure recurly.css is included in the build
+import recurlyCSS from './lib/recurly.css';
```

Comment:
This is just for build organization. The CSS file must be included in the build in order to be processed, and it really should not have been alongside the class definition.